index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:100,200,300,400,500,700"
+    />
+  </head>
+  <body>
+    <!-- navbar -->
+    <nav class="nav">
+      <a href="index.html" class="logo-link"> AOI </a>
+      <ul class="nav-list">
+        <li>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="currentColor"
+            class="bi bi-list"
+            viewBox="0 0 16 16"
+          >
+            <path
+              fill-rule="evenodd"
+              d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"
+            />
+          </svg>
+        </li>
+      </ul>
+    </nav>
+    <!-- intro section or header -->
+    <header id="intro-header">
+      <section id="intro">
+        <h1 class="greetings">I’m Abraham, Glad to see you!</h1>
+        <p class="intro-text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me. LET’S CONNECT
+        </p>
+        <p class="connect">LET’S CONNECT</p>
+        <div class="connect-links">
+          <a href="#">
+            <img
+              src="/Assets/Icons/twitter-vector.svg"
+              alt=""
+              class="social-icons"
+            />
+          </a>
+          <a href="#">
+            <img
+              src="/Assets/Icons/linkedin-vector.svg"
+              alt=""
+              class="social-icons"
+            />
+          </a>
+          <a href="#">
+            <img
+              src="/Assets/Icons/medium-vector.svg"
+              alt=""
+              class="social-icons"
+            />
+          </a>
+          <a href="#">
+            <img
+              src="/Assets/Icons/github-vector.svg"
+              alt=""
+              class="social-icons"
+            />
+          </a>
+          <a href="#">
+            <img
+              src="/Assets/Icons/social-icon-vector.svg"
+              alt=""
+              class="social-icons"
+            />
+          </a>
+        </div>
```

Comment:
Let's wrap the list of icons inside a `ul` tag. Also, the icons are broken as you can see in the provided snapshot. Please fix them.
![broken](https://user-images.githubusercontent.com/33268927/170671239-0a8348e4-408c-41a1-8ad3-558083a3ca7f.png)

Reply:
I have fixed this, Thanks.